su-template/default/layout/html/headers.erb

```diff
@@ -0,0 +1,15 @@
+  <meta charset="<%= charset %>">
+  <!-- VIEWPORT -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- AUTHOR and GENERATOR -->
+    <meta name="author" content="SketchUp Extensibility Team">
+    <meta name="generator" content="YARD <%= YARD::VERSION %> (ruby-<%= RUBY_VERSION %>) http://yardoc.org">
```

Comment:
Well it was just a thought. If it causes more trouble than it's worth, we'll change the generator to remove the version stamps.